third_party/3/waitress/adjustments.pyi

```diff
@@ -0,0 +1,60 @@
+from .compat import HAS_IPV6, PY2, WIN, string_types
+from .proxy_headers import PROXY_HEADERS
+from socket import SocketType
+from typing import Any, Mapping, Sequence, Optional, Set, Tuple, Union
+
+truthy: frozenset
+KNOWN_PROXY_HEADERS: frozenset
+
+def asbool(s: Optional[Union[bool, str, int]]) -> bool: ...
+def asoctal(s: str) -> int: ...
+def aslist_cronly(value: str) -> Sequence[str]: ...
+def aslist(value: str) -> Sequence[str]: ...
+def asset(value: str) -> Set[str]: ...
+def slash_fixed_str(s: str) -> str: ...
+def str_iftruthy(s: str) -> Optional[str]: ...
+def as_socket_list(sockets: Sequence[Any]) -> Sequence[SocketType]: ...
+
+class _str_marker(str): ...
+class _int_marker(int): ...
+class _bool_marker: ...
+
+class Adjustments:
+    host: _str_marker = ...
+    port: _int_marker = ...
+    listen: Sequence[str] = ...
+    threads: int = ...
+    trusted_proxy: Optional[str] = ...
+    trusted_proxy_count: Optional[int] = ...
+    trusted_proxy_headers: Set[str] = ...
+    log_untrusted_proxy_headers: bool = ...
+    clear_untrusted_proxy_headers: Union[_bool_marker, bool] = ...
+    url_scheme: str = ...
+    url_prefix: str = ...
+    ident: str = ...
+    backlog: int = ...
+    recv_bytes: int = ...
+    send_bytes: int = ...
+    outbuf_overflow: int = ...
+    outbuf_high_watermark: int = ...
+    inbuf_overflow: int = ...
+    connection_limit: int = ...
+    cleanup_interval: int = ...
+    channel_timeout: int = ...
+    log_socket_errors: bool = ...
+    max_request_header_size: int = ...
+    max_request_body_size: int = ...
+    expose_tracebacks: bool = ...
+    unix_socket: Optional[str] = ...
+    unix_socket_perms: int = ...
+    socket_options: Sequence[Tuple[int, int, int]] = ...
+    asyncore_loop_timeout: int = ...
+    asyncore_use_poll: bool = ...
+    ipv4: bool = ...
+    ipv6: bool = ...
+    sockets: Sequence[SocketType] = ...
+    def __init__(self, **kw: str) -> None: ...
+    @classmethod
+    def parse_args(cls, argv: str) -> Tuple[Mapping[str, Any], Any]: ...
+    @classmethod
+    def check_sockets(cls, sockets: Sequence[SocketType]) -> None: ...
```

Comment:
As `sockets` is only iterated over, this can use the more lenient `Iterable`.

```suggestion
    def check_sockets(cls, sockets: Iterable[SocketType]) -> None: ...
```